hyp3_gamma/insar/unwrapping_geocoding.py

```diff
@@ -97,24 +108,63 @@ def get_neighbors(array, i, j, n=1):
     return array[i_start:i_stop, j_start:j_stop]
 
 
-def ref_point_with_max_cc(data_cc: np.array, window_size=1):
-    '''
-    shift determine the window size, n=1 9-pixel window, n=2, 25-pixel window, etc.
-    '''
+def find_ref_point_with_largest_cc(data_cc: np.ndarray, indices: (np.array, np.array),
+                                   window_size, start_idx, pick_num):
 
-    data_cc_max = data_cc[data_cc < 1.0].max()
-    idx = np.where(data_cc == data_cc_max)
+    # get the indices of the  pixels with the highest coherence values
```

Comment:
Should "pixel" and "coherence value" be singular here?
```suggestion
    # get the indices of the pixel with the highest coherence value

```